deploy/deploy-k8s.yml

```diff
@@ -1,4 +1,24 @@
 ---
+- name: Install extra packages
+  hosts: all
+  become: true
+  tasks:
+
+  - name: Install packages
+    command: "rpm-ostree install {{ item }}"
+    with_items:
+      # socat is needed for Helm
+      - socat
```

Comment:
I had a thought. It may be a good idea to get this change in to Kubespray itself. Kubespray supports deploying helm on the kubernetes clusters it creates, so it should do this as well.

Reply:
kubernetes-incubator/kubespray#2879 is already opened for this and you've already found it. So this is okay till that gets fixed.